.pre-commit-config.yaml

```diff
@@ -35,7 +35,6 @@ repos:
           - build==1.0.0
           - pyproject_hooks==1.0.0
           - pytest==7.4.2
-        language_version: python3.8
```

Comment:
I don't think this should be a part of this PR. If we want to merge a few PRs simultaneously, it's best to make a cumulative PR with a real/natural merge commit combining several branches into `main` so that merging would pick up all the PRs individually.

Reply:
that's fine. my goal here was to get CI passing again, which included this change. it is currently brought in with a merge, but i'm happy to rebase once the other pr is in.